src/Integrals.jl

```diff
@@ -38,7 +38,8 @@ function scale_x(ub, lb, x)
 end
 
 function v_inf(t)
-    return t ./ (1 .- t .^ 2)
+    t2 = t .^ 2
+    return t ./ (1 .- t2)
```

Comment:
This, seemingly irrelevant change, is required to make the transformed function type stable with StaticVectors.

Reply:
Or just ` map(t->t / (1 - t ^ 2), t)`. SA's broadcast is really bad.